src/routes/cards/[cardsName]/+page.svelte

```diff
@@ -23,9 +23,9 @@
   }
 </script>
 
-<div class="add-wallet mobile">
-  <a href={pkpass}><img class="ios" src={iosWallet} alt="button" /></a>
-  <a href={cardData.passGoogle}><img class="android" src={googleWallet} alt="button" /></a>
+<div class="add-wallet">
+  <a href={pkpass}><img class="image-button" src={iosWallet} alt="button" /></a>
```

Comment:
image-button is bad class name, it's not a button

alt should be more descriptive